plugins/@grouparoo/mongo/src/lib/table-import/getColumns.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  GetColumnDefinitionsMethod,
+  ColumnDefinitionMap,
+  FilterOperation,
+  ColumnType,
+} from "@grouparoo/app-templates/dist/source/table";
+
+export const getColumns: GetColumnDefinitionsMethod = async ({
+  connection,
+  tableName,
+  appOptions,
+  sourceOptions,
+}) => {
+  const map: ColumnDefinitionMap = {};
+  if (sourceOptions) {
+    const fields = sourceOptions.fields.toString().split(",");
```

Comment:
let's also `trim`: `sourceOptions.fields.toString().split(",").map(f => f.trim())`

Reply:
And have a test with spaces.